Sources/MapboxNavigation/DateComponentsFormatter+NavigationAdditions.swift

```diff
@@ -21,4 +21,14 @@ extension DateComponentsFormatter {
         formatter.allowedUnits = [.day, .hour, .minute]
         return formatter
     }()
+    
+    public static func travelTimeString(_ interval: TimeInterval,
+                                        signed: Bool,
+                                        unitStyle: DateComponentsFormatter.UnitsStyle?) -> String {
+        let formatter = DateComponentsFormatter()
+        formatter.unitsStyle = unitStyle ?? (interval < 3600 ? .short : .abbreviated)
+        let timeString = formatter.string(from: signed ? interval : abs(interval)) ?? ""
+        
+        return signed ? "\(interval >= 0 ? "+":"")\(timeString)" : timeString
```

Comment:
Use `NumberFormatter.plusSign` instead of a literal plus sign. In theory, this property respects the formatter’s current locale, though I don’t think it differs in any locales at the moment.

Reply:
But it's `DateComponentsFormatter` so it does not provide access to `NumberFormatter` and does not provide similar formatting options either.

Reply:
Oh yeah, that’s annoying. I guess we could do `formatter.calendar?.locale?.plusSign`, but that’s kinda funky; I have no idea if the Calendar would be set up with the correct number formatter for time arithmetic in general. How about making a `+%@` localizable string instead?

Reply:
Added. I'd say that we won't ever need it, but it shouldn't be a problem to have it too